randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -59,19 +53,27 @@ Extra - save_index: 0
               Speed Booster and Shinespark (Beginner)
   > Door to Parlor
       Trivial
+  > Door to Crateria Power Bomb Room
+      Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 79
+  > Door to Crateria Tube
+      Screw Attack or Speed Booster or Can Lay Any Bombs
+  > Event - Win Game
+      After Defeat Mother Brain
+  > Top Pillar
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 50
```

Comment:
IBJ missing

Reply:
These always feel strange to me. If someone can IBJ, they can definitely do beginner wall jumps. But I've added for the sake of formalism.

Reply:
Some people simply might not want to IBJ though for their seeds for various reasons. And doing "you can do trick X, so we'll now silently expect you to do trick Y" is a very bad thing.